src/components/elementView/powerView/bladePowerView.tsx

```diff
@@ -289,9 +308,13 @@ export class BladePowerView extends React.PureComponent<
         this.props.asset.chassis.model &&
         this.props.asset.chassis.model.vendor &&
         this.isBladePowerNetworkControlled()
-          ? this.state.statusLoaded
+          ? this.isBladeInStorage()
+            ? this.renderInStorageCallout()
+            : this.state.statusLoaded
             ? this.renderPowerTable(false)
             : this.renderPowerTable(true)
+          : this.isBladeInStorage()
```

Comment:
not beautiful logic but had to add this case if it's a blade in storage not inside a chassis